code/modules/lavaland/lavaland_theme.dm

```diff
@@ -0,0 +1,55 @@
+/datum/lavaland_theme
+	/// Name of lavaland theme
+	var/name = "Not Specified"
+	/// Typepath of turf the `/turf/simulated/floor/lava/mapping_lava` will be changed to on Late Initialization
+	var/turf/simulated/floor/primary_turf_type = /turf/simulated/floor/lava/lava_land_surface
```

Comment:
This should be null by default, so you can nullcheck it like this
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/75fa38c1-c57f-45b4-a1ea-883adc50072b)

```suggestion
	var/turf/simulated/floor/primary_turf_type
```